### PR TITLE
sgdboop: init at 1.2.8

### LINKED ISSUE #220937: Package request: SGDBoop
**Project description**

This project makes it easy to add third party games and software to Steam, with Artwork. Designed for the SteamDeck, it should work fine on Desktop Steam as well.

**Metadata**

* homepage URL: https://www.steamgriddb.com/boop
* source URL: https://github.com/SteamGridDB/SGDBoop
* license: [CC-BY-NC-SA-4.0](https://github.com/SteamGridDB/SGDBoop/blob/main/LICENSE)
* platforms: Linux

### PR DESCRIPTION
**Pull request by puffnfresh**
## Description of changes

I haven't tested the browser integration but I think stuff like this works:

```
SGDBoop 'sgdb://boop/grid/74165/nonsteam'
```

Fixes #220937.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Saturn745** [comment]
Just tested this and everything works fine. I was able to add assets to steam with the URL protocol. Does anything need to be done for this to get merged?

**Comment by marcusramberg** [comment]
@ofborg eval

**Comment by marcusramberg** [comment]
@ofborg eval

**Comment by marcusramberg** [comment]
@ofborg test editorconfig

File: pkgs/development/libraries/iup/default.nix [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+, stdenv
+, fetchurl
+
+, lsb-release
+, which
+, pkg-config
+, gtk3
+, cd
+, libGLU
+, ftgl
```
Comment (marcusramberg) [inline pkgs/development/libraries/iup/default.nix:11]: Trailing whitespace here fails the editorconfig check.

**Comment by KucharczykL** [comment]
Hi, was this closed by accident? Would be nice to have it!

**Pull request closed without merging**